modules/ROOT/pages/data-sync.adoc

```diff
@@ -2,8 +2,38 @@
 include::_partials/attributes.adoc[]
 // end {partialsdir}/attributes.adoc[]
 
-include::_partials/sync/index.adoc[leveloffset=0]
-[[setup]]
+:service-name: Data Sync
+
+//':context:' is a vital parameter. See: http://asciidoctor.org/docs/user-manual/#include-multiple
+:context: sync
+
+[id='{context}']
+= Developing a {sync-service} App
+:toc:
+
+
+
+Unlike other AeroGear mobile services which provide a server and an API, Data Sync is a library that is used to develop services. Typically, you develop a {sync-service} service as follows:
+
+* Design a link:https://graphql.org/learn[GraphQL] schema
+* Develop a server side service and mobile application
```

Comment:
should this just be `Develop a server side service` and the mobile application would be a separate step?